src/main/java/com/wolox/training/controllers/UserController.java

```diff
@@ -0,0 +1,87 @@
+package com.wolox.training.controllers;
+
+import com.wolox.training.exceptions.BookNotFoundException;
+import com.wolox.training.exceptions.UserIdMismatchException;
+import com.wolox.training.exceptions.UserNotFoundException;
+import com.wolox.training.models.Book;
+import com.wolox.training.models.User;
+import com.wolox.training.repositories.BookRepository;
+import com.wolox.training.repositories.UserRepository;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+
+    private final UserRepository userRepository;
+
+    private final BookRepository bookRepository;
+
+    public UserController(UserRepository userRepository, BookRepository bookRepository) {
+        this.userRepository = userRepository;
+        this.bookRepository = bookRepository;
+    }
```

Comment:
You can avoid this constructor by using the `@Autorwired` annotation. Check the 3.1. section in [this](https://www.baeldung.com/spring-autowire) link

Reply:
The linter warned me to prefer to use it this way

Reply:
already made the changes